src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.WaitForChanged.cs

```diff
@@ -124,6 +124,57 @@ public void NonZeroTimeout_NoActivity_TimesOut(WatcherChangeTypes changeType, bo
             }
         }
 
+        [Theory]
+        [InlineData(false)]
```

Comment:
these FSW tests are inherently flaky. these tests are 99% duplicating the existing tests (of course) and so adding more opportunities for them to flake out.
..
Since we know the choice of timespan vs int and the other variables do not relate to each other in any way, you can add coverage without duplicating. Just take mix in another parameter into the InlineData's that tells it whether to use a timespan or not. eg
```c#
        [Theory]
        [InlineData(false, false)]
        [InlineData(true, true)]
        public void NonZeroTimeout_NoEvents_TimesOut(bool enabledBeforeWait, bool useTimeSpan)
        {
...
                 if (useTimeSpan)
                 {
                          AssertTimedOut(fsw.WaitForChanged(WatcherChangeTypes.All,  TimeSpan.FromMilliseconds(0)));
                 }
                 else
...
```